mappings/net/minecraft/data/SnbtProvider.mapping

```diff
@@ -19,10 +28,12 @@ CLASS net/minecraft/class_2463 net/minecraft/data/SnbtProvider
 		FIELD field_20538 name Ljava/lang/String;
 		FIELD field_20539 bytes [B
 		FIELD field_20540 sha1 Ljava/lang/String;
+		FIELD field_24616 jsonizedContent Ljava/lang/String;
 		METHOD <init> (Ljava/lang/String;[BLjava/lang/String;Ljava/lang/String;)V
 			ARG 1 name
 			ARG 2 bytes
-			ARG 3 sha1
+			ARG 3 jsonizedContent
```

Comment:
Is there a better name for this? I am not a fan of `jsonizedContent` because 'jsonized' is not an actual word.

Reply:
Just `text` or `content` would be better as those are just written to a file.
```suggestion
			ARG 3 text
```
Gradle calls these `text`.

Reply:
The here is the NBT structure converted to it's json form, so I believe "jsonized" is an accurate descriptor. Certainly better than "text"

Reply:
Is it JSON or SNBT (string NBT)? Those two formats are similar but different, and this might be SNBT since it's an SNBT class.

Reply:
It is SNBT but I'm not really familiar with SNBT and I think calling it `snbtContent` would create more confusion that just saying it's json.

Reply:
Alrighty. I'll use `snbtContent` then, unless/until anyone else objects.